pkg/skaffold/util/util.go

```diff
@@ -227,3 +229,15 @@ func AbsFile(workspace string, filename string) (string, error) {
 	}
 	return filepath.Abs(file)
 }
+
+// NonEmptyLines scans the provided input and returns the non-empty strings found as an array
+func NonEmptyLines(input []byte) []string {
+	var result = []string{}
```

Comment:
`var result []string`

Reply:
That results in a nil element being returned when there are 0 lines.  Coding against that seemed awkward.